Package.swift

```diff
@@ -4,6 +4,7 @@ import PackageDescription
 
 let package = Package(
     name: "XcodeGen",
+    platforms: [.macOS(.v10_13)],
```

Comment:
It should be 10.13. Because `.sortedKey` option is used on `FileCache` 
https://github.com/yonaskolb/XcodeGen/blob/master/Sources/XcodeGenKit/CacheFile.swift#L17